manageiq-operator/pkg/apis/manageiq/v1alpha1/manageiq_types.go

```diff
@@ -12,286 +12,334 @@ import (
 	// +optional
-	HttpdMemoryRequest string `json:"httpdMemoryRequest"`
+	KafkaSecret string `json:"kafkaSecret,omitempty"`
+
+	// Kafka volume size (default: 1Gi)
+	// +optional
+	KafkaVolumeCapacity string `json:"kafkaVolumeCapacity,omitempty"`
+
+	// Memcached deployment CPU limit (default: no limit)
+	// +optional
+	MemcachedCpuLimit string `json:"memcachedCpuLimit,omitempty"`
+
+	// Memcached deployment CPU request (default: no request)
+	// +optional
+	MemcachedCpuRequest string `json:"memcachedCpuRequest,omitempty"`
 
 	// Image string used for the memcached deployment
 	// (default: <MemcachedImageName>:<MemcachedImageTag>)
 	// +optional
-	MemcachedImage string `json:"memcachedImage"`
+	MemcachedImage string `json:"memcachedImage,omitempty"`
+
 	// Image used for the memcached deployment (default: manageiq/memcached)
 	// +optional
-	MemcachedImageName string `json:"memcachedImageName"`
+	MemcachedImageName string `json:"memcachedImageName,omitempty"`
+
 	// Image tag used for the memcached deployment (default: latest)
 	// +optional
-	MemcachedImageTag string `json:"memcachedImageTag"`
+	MemcachedImageTag string `json:"memcachedImageTag,omitempty"`
 
-	// Memcached deployment CPU limit (default: no limit)
+	// Memcached max simultaneous connections (default: 1024)
 	// +optional
-	MemcachedCpuLimit string `json:"memcachedCpuLimit"`
-	// Memcached deployment CPU request (default: no request)
+	MemcachedMaxConnection string `json:"memcachedMaxConnection,omitempty"`
+
+	// Memcached item memory in megabytes (default: 64)
 	// +optional
-	MemcachedCpuRequest string `json:"memcachedCpuRequest"`
+	MemcachedMaxMemory string `json:"memcachedMaxMemory,omitempty"`
+
 	// Memcached deployment memory limit (default: no limit)
 	// +optional
-	MemcachedMemoryLimit string `json:"memcachedMemoryLimit"`
+	MemcachedMemoryLimit string `json:"memcachedMemoryLimit,omitempty"`
+
 	// Memcached deployment memory request (default: no limit)
 	// +optional
-	MemcachedMemoryRequest string `json:"memcachedMemoryRequest"`
+	MemcachedMemoryRequest string `json:"memcachedMemoryRequest,omitempty"`
 
-	// Memcached max simultaneous connections (default: 1024)
+	// Memcached max item size (default: 1m, min: 1k, max: 1024m)
 	// +optional
-	MemcachedMaxConnection string `json:"memcachedMaxConnection"`
-	// Memcached item memory in megabytes (default: 64)
+	MemcachedSlabPageSize string `json:"memcachedSlabPageSize,omitempty"`
+
+	// Secret containing the trusted CA certificate file(s) for the OIDC server.
+	// Only used with the openid-connect authentication type
 	// +optional
-	MemcachedMaxMemory string `json:"memcachedMaxMemory"`
-	// Memcached max item size (default: 1m, min: 1k, max: 1024m)
+	OIDCCACertSecret string `json:"oidcCaCertSecret,omitempty"`
+
+	// Secret name containing the OIDC client id and secret
+	// Only used with the openid-connect authentication type
+	// +optional
+	OIDCClientSecret string `json:"oidcClientSecret,omitempty"`
```

Comment:
This name _always_ confuses it, because it's a kube secret _NAME_, which that secret contains the client id and client secret